### PR TITLE
fix(plugin-text): clear timeouts in useEffect hooks

### PR DESCRIPTION
**Pull request by hejtful**
Found some timeout that are not cleared on unmount. Although it probably isn't the cause of the performance bug, it's still good to clear those timeouts.

### CODE REVIEW

File: src/serlo-editor-repo/plugin-text/components/text-editor.tsx [hunk 1]
```diff
@@ -98,8 +98,10 @@ export function TextEditor(props: TextEditorProps) {
 
   useEffect(() => {
     if (focused === false) return
+
     // ReactEditor.focus(editor) does not work without being wrapped in setTimeout
     // See: https://stackoverflow.com/a/61353519
+    let timeout: ReturnType<typeof setTimeout>
     setTimeout(() => {
```
Comment (Entkenntnis) [inline src/serlo-editor-repo/plugin-text/components/text-editor.tsx:105]: `timeout =` missing? :)

Comment (Entkenntnis) [inline src/serlo-editor-repo/plugin-text/components/text-editor.tsx]: `timeout =` missing? :)

**Review by Entkenntnis — CHANGES_REQUESTED** [review]

File: src/serlo-editor-repo/plugin-text/components/text-editor.tsx (hunk 1 above)
Comment (hejtful) [reply src/serlo-editor-repo/plugin-text/components/text-editor.tsx]: 😅 Thanks!!

**Review by Entkenntnis — APPROVED** [review]
thx

**Pull request merged**